src/Orleans.Runtime/Hosting/Clustering/IClusterProvider.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Orleans.Runtime.Hosting.Clustering
+{
+    public interface IClusterProvider
```

Comment:
This interface is currently duplicated (here and in the Kubernetes hosting package)

Reply:
Yes, I have forgotten to delete it.